systemtests/tests/py3plug-fd-local-fileset-basic/testrunner-basic

```diff
@@ -23,70 +23,57 @@ JobName=backup-bareos-fd
 
 #shellcheck source=../../scripts/functions
 . "${rscripts}"/functions
-"${rscripts}"/cleanup
-"${rscripts}"/setup
 
-# Fill ${BackupDirectory} with data.
-setup_data
-
-# this test does not work with links, because
-# python 2 does not support setting mtime on links
-# they are restored properly but with wrong timestamps
-# This is supposed to work with the future plugin-fd interface
-# using Python3
-find ${tmp}/data/weird-files -type l -exec rm {} \;
-find tmp/data/weird-files -links +1 -type f -exec rm {} \;
-
-grep py3 <<<"$TestName" && rm tmp/data/weird-files/filename-with-non-utf8-bytestring*
-
-print_debug "$(locale)"
+if [ -f "${working}/${TestName}"-fd.trace ]; then
+  rm "${working}/${TestName}"-fd.trace
+fi
 
 start_test
 
 cat <<END_OF_DATA >$tmp/bconcmds
 @$out ${NULL_DEV}
 messages
-@$out $tmp/log1.out
+@$out ${log_home}/log1.out
+setdebug level=800 client trace=0
 setdebug level=800 client trace=1
-label volume=TestVolume001 storage=File pool=Full
-run job=$JobName yes
+run job=$JobName level=Full yes
 status director
 status client
 status storage=File
 wait
 messages
 END_OF_DATA
 
-run_bareos "$@"
-if is_debug; then
-  cat ${working_dir}/*.trace
-fi
+run_bconsole "$@"
 check_for_zombie_jobs storage=File
-stop_bareos
 
 cat <<END_OF_DATA >$tmp/bconcmds
 @#
 @# now do a restore
 @#
-@$out $tmp/log2.out
-restore client=bareos-fd fileset=PluginTest where=$tmp/bareos-restores select all done yes
+@$out ${log_home}/log2.out
+restore client=bareos-fd fileset=PluginTest where=${log_home}/bareos-restores select all done yes
```

Comment:
The name `log_home` is misleading as this seems to be a testrunner-individual temporary directory?
How about `runner_tmp`